openlibrary/core/admin.py

```diff
@@ -101,6 +101,36 @@ def get_counts(self, ndays=28, times=False):
             return Stats.get_counts(self, ndays, times)
 
 
+class VisitorStats(Stats):
+    """
+    Read the unique visitors (IP addresses) per day for the last ndays from graphite.
+    """
+
+    @cache.method_memoize
+    def _get_graphite_data(self, ndays: int) -> list[int]:
```

Comment:
This cache will be a problem. It'll be cached for the lifetime of the runtime; potentially multiple days. We want it to only be a few hours. Maybe use a memcache backed cache.

Reply:
What about like line 134 below?
https://github.com/internetarchive/openlibrary/pull/7745/files/d008e4fd6385e1a89efdaa53419f66add2a5c346#diff-347744051e2ee8306ebd59f16efde4121b8c7b8348cd78cbfd2ad36331289075R134

Reply:
That seems perfect! 5min expiry sounds good.